vignettes/userguide.Rmd

```diff
@@ -189,6 +189,42 @@ options(
 
 We recommend that you [set up SSH keys](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server) for password-less login.
 
+### Options
+
+The various configurable options are mentioned throughout the documentation where applicable, however, we list all of the options here for reference.
+
+Options can be set by including a call to `options()` in your `.Rprofile`, or by calling `options()` in a script or interactively during a session.
+
+- `clustermq.scheduler` 
+  + One of the supported `clustermq` schedulers; options are `"LOCAL"`, `"multiprocess"`, `"multicore"`, `"lsf"`, `"sge"`, `"slurm"`, `"pbs"`, `"Torque"`, or `"ssh"` (default is `qsys_default`, which is usually `"LOCAL"`)
+
+- `clustermq.host` 
+  + The name of the node or device for constructing the `ZeroMQ` host address (default is `Sys.info()["nodename"]`)
+
+- `clustermq.ssh.host`
+  + The user name and host for connecting to the HPC via SSH (e.g. `user@host`); we recommend setting up SSH keys for password-less login.
+
+- `clustermq.ssh.log`
+  + Path for a file that will be created and populated with logging information regarding the SSH connection (e.g. `"~/cmq_ssh.log"`); helpful for debugging purposes
```

Comment:
this log file is created on the ssh host, not locally (mention to void confusion)